src/types/GlyphWidth.cpp

```diff
@@ -5,16 +5,14 @@
 #include "inc/CodepointWidthDetector.hpp"
 #include "inc/GlyphWidth.hpp"
 
-#pragma warning(suppress : 26426)
-// TODO GH 2676 - remove warning suppression and decide what to do re: singleton instance of CodepointWidthDetector
-static CodepointWidthDetector widthDetector;
-
 // Function Description:
 // - determines if the glyph represented by the string of characters should be
 //      wide or not. See CodepointWidthDetector::IsWide
 bool IsGlyphFullWidth(const std::wstring_view& glyph) noexcept

```

Comment:
does Terminal use this function? IF NOT, we can move all off GlyphWidth.cpp into host and remove it from the surface of types.

Reply:
My goal is to get rid of it entirely within 1.22 so I think it's fine if it stays here until then.

Reply:
OK!